### ISSUE TITLE
maxfiltered data as raw or derivatives

### ISSUE THREAD

**Issue by robertoostenveld** [bids-standard/bids-specification#23]
There is conflicting information as to whether maxfiltered MEG data should be represented as as raw (with proc- or as derivatives

The main specification now contains 

> After applying the MaxFilter pre-processing tool, files should be renamed with the corresponding label (e.g. proc-sss) and placed into a derivatives subfolder.

but also 

> SoftwareFilters | REQUIRED. List of temporal and/or spatial software filters applied, or ideally key:value pairs of pre-applied software filters and their parameter values: e.g., {"SSS": {"frame": "head", "badlimit": 7}} ...

where the last relates to the meg.json of the raw data. This is in line with "8.4.1 MEG recording data" which gives the example that "proc-xxx" with xxx=sss, tsss, trans, quat, mc, etcetera, can be used for a maxfiltered meg dataset.

**Comment by jasmainak** [comment]
I agree this is confusing. I think Maxfiltered files should be considered derivatives unless it is done during the acquisition itself (and thus we do not have access to the raw files). If they are applied afterwards, one could process the files using different parameters (just like in temporal filtering) which could lead to different variants of the processed file. 

While this is not as common, we should perhaps allow for this possibility? In the derivatives specification, we have a keyword called `desc` which allows a sidecar json with the different processing parameters specified. What do you think? If you agree, I could flesh out a proposal for Maxfiltered files in the derivatives specification.
